app/assets/javascripts/initializers/initNotifications.js

```diff
@@ -114,46 +122,38 @@ function onClickReaction(event) {
     });
 }
 
+function onClickReply(event) {
+  event.preventDefault();
+  var thisButt = event.target;
+  document
+    .getElementById('comment-form-for-' + thisButt.dataset.reactableId)
+    .classList.add('showing');
+  thisButt.innerHTML = '';
+  setTimeout(() => {
+    document
+      .getElementById('comment-textarea-for-' + thisButt.dataset.reactableId)
+      .focus();
+  }, 30);
+}
+
 function initReactions() {
   setTimeout(() => {
     if (document.getElementById('notifications-container')) {
-      var butts = document.getElementsByClassName('reaction-button');
-      var i;
-      var butt;
-      for (i = 0; i < butts.length; i += 1) {
-        butt = butts[i];
-        butt.onclick = event => onClickReaction(event);
-      }
-      butts = document.getElementsByClassName('toggle-reply-form');
-      for (i = 0; i < butts.length; i += 1) {
-        butt = butts[i];
-        butt.onclick = event => {
-          event.preventDefault();
-          var thisButt = this;
-          document
-            .getElementById('comment-form-for-' + thisButt.dataset.reactableId)
-            .classList.add('showing');
-          thisButt.innerHTML = '';
-          setTimeout(() => {
-            document
-              .getElementById(
-                'comment-textarea-for-' + thisButt.dataset.reactableId,
-              )
-              .focus();
-          }, 30);
-        };
-      }
+      addOnClickHandlerToButtons('reaction-button', onClickReactionButton);
+      addOnClickHandlerToButtons('toggle-replay-form', onClickReply);
```

Comment:
@maestromac I think the bug is because of this typo here, should be toggle-reply-form I'll fix it.